higher_health/forms.py

```diff
@@ -202,7 +202,8 @@ def __init__(self, *args, **kwargs):
                     response = requests.get(
                         f"https://maps.googleapis.com/maps/api/place/details/json?{querystring}"
                     )
-                    geometry = response.json()["result"]["geometry"]["location"]
+                    place_details = response.json()
+                    geometry = place_details["result"]["geometry"]["location"]
```

Comment:
Why not use a `.get()` for result?

Reply:
A `.get` will return `None` if the `result` key isn't in the response, which means the `["geometry"]` lookup will fail.

We could chain gets, `.get("result", {}).get("geometry", {}).get("location")`, but then `geometry` could be None, in which case we need to fail anyway.

Doing it this way is a "assert that the response is in this format, or fail". Once we see what response we're getting, we could wrap this is a `try` block, and handle the failure more gracefully for certain errors.

Reply:
I guess my point is that this will not make the errors go away.